.github/workflows/build_tests.yml

```diff
@@ -19,10 +19,15 @@ jobs:
           python -m pip install build wheel
       - name: Install System Dependencies
         run: |
-          sudo apt install python3-dev swig libssl-dev libfann-dev portaudio19-dev
+          sudo apt-get update
+          sudo apt install python3-dev swig libssl-dev libfann-dev portaudio19-dev libpulse-dev
       - name: Build Distribution Packages
         run: |
           python setup.py bdist_wheel
+      - name: Install tflite_runtime workaround tflit bug
```

Comment:
Does this deserve an issue?

Reply:
maybe, there is one in the upstream tflit repo already, happens in some platforms due to some mismatch with post release version of the wheel it tries to download or something during setup.py

Reply:
Just want to make sure we remember to remove this when upstream gets fixed, though it's not a big deal just for automation